### PR TITLE
Valid accounts   guest user invitation

### PR DESCRIPTION
**Pull request by tomer-pan**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://jira-dc.paloaltonetworks.com/browse/CIAC-12203

## Description
New playbook: Valid Accounts - Successful guest user invitation
## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/Core/Playbooks/playbook-Successful_guest_user_invitation_README.md [hunk 1]
```diff
@@ -0,0 +1,82 @@
+This playbook addresses the following alerts:
```
Comment (AdiPeret) [inline Packs/Core/Playbooks/playbook-Successful_guest_user_invitation_README.md:1]: This should be a section/title

File: Packs/Core/Playbooks/playbook-Successful_guest_user_invitation.yml [hunk 2]
```diff
@@ -0,0 +1,1604 @@
+id: Successful guest user invitation
+version: -1
+description: |-
+  **This playbook addresses the following alerts**:
```
Comment (ShirleyDenkberg) [inline Packs/Core/Playbooks/playbook-Successful_guest_user_invitation.yml:4]:
```suggestion
  **This playbook addresses the following alert**:
```

File: Packs/Core/Playbooks/playbook-Successful_guest_user_invitation.yml [hunk 3]
```diff
@@ -0,0 +1,1604 @@
...
+        - simple: No Action
+        - simple: Disable Invited User
+        - simple: Disable Inviting User
+        - simple: Disable Both Users
+        fieldassociated: ""
+        placeholder: ""
+        tooltip: ""
+        readonly: false
+      title: Analyst Action
+      description: Analyst Review
```
Comment (ShirleyDenkberg) [inline Packs/Core/Playbooks/playbook-Successful_guest_user_invitation.yml:549]:
```suggestion
      description: Analyst review.
```

File: Packs/Core/Playbooks/playbook-Successful_guest_user_invitation.yml [hunk 4]
```diff
@@ -0,0 +1,1604 @@
...
+    task:
+      id: d05451a6-6c9e-40a4-8498-3655c8540813
+      version: -1
+      name: Get malicious Email value
+      description: |-
+        Set a value in context under the key you entered. If no value is entered, the script doesn't do anything.
+
+        This automation runs using the default Limited User role, unless you explicitly change the permissions.
+        For more information, see the section about permissions here:
+        https://docs-cortex.paloaltonetworks.com/r/Cortex-XSOAR/6.10/Cortex-XSOAR-Administrator-Guide/Automations
```
Comment (ShirleyDenkberg) [inline Packs/Core/Playbooks/playbook-Successful_guest_user_invitation.yml:570]:
```suggestion
     - For Cortex XSOAR 6, see https://docs-cortex.paloaltonetworks.com/r/Cortex-XSOAR/6.x/Cortex-XSOAR-Playbook-Design-Guide/Automations
     - For Cortex XSOAR 8 Cloud, see https://docs-cortex.paloaltonetworks.com/r/Cortex-XSOAR/8/Cortex-XSOAR-Cloud-Documentation/Create-a-script
     - For Cortex XSOAR 8 On-prem, see https://docs-cortex.paloaltonetworks.com/r/Cortex-XSOAR/8.7/Cortex-XSOAR-On-prem-Documentation/Create-a-script
```

File: Packs/Core/Playbooks/playbook-Successful_guest_user_invitation.yml [hunk 5]
```diff
@@ -0,0 +1,1604 @@
...
+    task:
+      id: b6d9862d-d090-4d8b-8e17-55a8ba786a55
+      version: -1
+      name: Get risky user value
+      description: |-
+        Set a value in context under the key you entered. If no value is entered, the script doesn't do anything.
+
+        This automation runs using the default Limited User role, unless you explicitly change the permissions.
+        For more information, see the section about permissions here:
+        https://docs-cortex.paloaltonetworks.com/r/Cortex-XSOAR/6.10/Cortex-XSOAR-Administrator-Guide/Automations
```
Comment (ShirleyDenkberg) [inline Packs/Core/Playbooks/playbook-Successful_guest_user_invitation.yml:632]:
```suggestion
        - For Cortex XSOAR 6, see https://docs-cortex.paloaltonetworks.com/r/Cortex-XSOAR/6.x/Cortex-XSOAR-Playbook-Design-Guide/Automations
        - For Cortex XSOAR 8 Cloud, see https://docs-cortex.paloaltonetworks.com/r/Cortex-XSOAR/8/Cortex-XSOAR-Cloud-Documentation/Create-a-script
       - For Cortex XSOAR 8 On-prem, see https://docs-cortex.paloaltonetworks.com/r/Cortex-XSOAR/8.7/Cortex-XSOAR-On-prem-Documentation/Create-a-script
```

File: Packs/Core/Playbooks/playbook-Successful_guest_user_invitation.yml [hunk 6]
```diff
@@ -0,0 +1,1604 @@
...
+    task:
+      id: 1e118a21-f5b8-4d92-8c86-06046d48a485
+      version: -1
+      name: Get risky user reasons value
+      description: |-
+        Set a value in context under the key you entered. If no value is entered, the script doesn't do anything.
+
+        This automation runs using the default Limited User role, unless you explicitly change the permissions.
+        For more information, see the section about permissions here:
+        https://docs-cortex.paloaltonetworks.com/r/Cortex-XSOAR/6.10/Cortex-XSOAR-Administrator-Guide/Automations
```
Comment (ShirleyDenkberg) [inline Packs/Core/Playbooks/playbook-Successful_guest_user_invitation.yml:717]:
```suggestion
        - For Cortex XSOAR 6, see https://docs-cortex.paloaltonetworks.com/r/Cortex-XSOAR/6.x/Cortex-XSOAR-Playbook-Design-Guide/Automations
        - For Cortex XSOAR 8 Cloud, see https://docs-cortex.paloaltonetworks.com/r/Cortex-XSOAR/8/Cortex-XSOAR-Cloud-Documentation/Create-a-script
       - For Cortex XSOAR 8 On-prem, see https://docs-cortex.paloaltonetworks.com/r/Cortex-XSOAR/8.7/Cortex-XSOAR-On-prem-Documentation/Create-a-script
```

File: Packs/Core/Playbooks/playbook-Successful_guest_user_invitation.yml [hunk 7]
```diff
@@ -0,0 +1,1604 @@
...
+    task:
+      id: 6c02e6b9-cd98-4865-8412-1c1bf2e0b401
+      version: -1
+      name: Extract Azure user detections
+      description: |-
+        Set a value in context under the key you entered. If no value is entered, the script doesn't do anything.
+
+        This automation runs using the default Limited User role, unless you explicitly change the permissions.
+        For more information, see the section about permissions here:
+        https://docs-cortex.paloaltonetworks.com/r/Cortex-XSOAR/6.10/Cortex-XSOAR-Administrator-Guide/Automations
```
Comment (ShirleyDenkberg) [inline Packs/Core/Playbooks/playbook-Successful_guest_user_invitation.yml:827]:
```suggestion
        - For Cortex XSOAR 6, see https://docs-cortex.paloaltonetworks.com/r/Cortex-XSOAR/6.x/Cortex-XSOAR-Playbook-Design-Guide/Automations
        - For Cortex XSOAR 8 Cloud, see https://docs-cortex.paloaltonetworks.com/r/Cortex-XSOAR/8/Cortex-XSOAR-Cloud-Documentation/Create-a-script
       - For Cortex XSOAR 8 On-prem, see https://docs-cortex.paloaltonetworks.com/r/Cortex-XSOAR/8.7/Cortex-XSOAR-On-prem-Documentation/Create-a-script
```

File: Packs/Core/Playbooks/playbook-Successful_guest_user_invitation.yml [hunk 8]
```diff
@@ -0,0 +1,1604 @@
...
+    task:
+      id: a890149a-86f7-4dd2-8c9c-9b8fbb03de03
+      version: -1
+      name: Extract Azure user alerts
+      description: |-
+        Set a value in context under the key you entered. If no value is entered, the script doesn't do anything.
+
+        This automation runs using the default Limited User role, unless you explicitly change the permissions.
+        For more information, see the section about permissions here:
+        https://docs-cortex.paloaltonetworks.com/r/Cortex-XSOAR/6.10/Cortex-XSOAR-Administrator-Guide/Automations
```
Comment (ShirleyDenkberg) [inline Packs/Core/Playbooks/playbook-Successful_guest_user_invitation.yml:901]:
```suggestion
        - For Cortex XSOAR 6, see https://docs-cortex.paloaltonetworks.com/r/Cortex-XSOAR/6.x/Cortex-XSOAR-Playbook-Design-Guide/Automations
        - For Cortex XSOAR 8 Cloud, see https://docs-cortex.paloaltonetworks.com/r/Cortex-XSOAR/8/Cortex-XSOAR-Cloud-Documentation/Create-a-script
       - For Cortex XSOAR 8 On-prem, see https://docs-cortex.paloaltonetworks.com/r/Cortex-XSOAR/8.7/Cortex-XSOAR-On-prem-Documentation/Create-a-script
```

File: Packs/Core/Playbooks/playbook-Successful_guest_user_invitation.yml [hunk 9]
```diff
@@ -0,0 +1,1604 @@
...
+    task:
+      id: d2afb6c7-6bdb-450c-801f-5c051fd4b93a
+      version: -1
+      name: Get user UPN
+      description: |-
+        Set a value in context under the key you entered. If no value is entered, the script doesn't do anything.
+
+        This automation runs using the default Limited User role, unless you explicitly change the permissions.
+        For more information, see the section about permissions here:
+        https://docs-cortex.paloaltonetworks.com/r/Cortex-XSOAR/6.10/Cortex-XSOAR-Administrator-Guide/Automations
```
Comment (ShirleyDenkberg) [inline Packs/Core/Playbooks/playbook-Successful_guest_user_invitation.yml:947]:
```suggestion
        - For Cortex XSOAR 6, see https://docs-cortex.paloaltonetworks.com/r/Cortex-XSOAR/6.x/Cortex-XSOAR-Playbook-Design-Guide/Automations
        - For Cortex XSOAR 8 Cloud, see https://docs-cortex.paloaltonetworks.com/r/Cortex-XSOAR/8/Cortex-XSOAR-Cloud-Documentation/Create-a-script
       - For Cortex XSOAR 8 On-prem, see https://docs-cortex.paloaltonetworks.com/r/Cortex-XSOAR/8.7/Cortex-XSOAR-On-prem-Documentation/Create-a-script
```

File: Packs/Core/Playbooks/playbook-Successful_guest_user_invitation.yml [hunk 10]
```diff
@@ -0,0 +1,1604 @@
...
+    task:
+      id: d29624dd-7417-474c-8a40-d7e5d03463c3
+      version: -1
+      name: Get malicious Domain value
+      description: |-
+        Set a value in context under the key you entered. If no value is entered, the script doesn't do anything.
+
+        This automation runs using the default Limited User role, unless you explicitly change the permissions.
+        For more information, see the section about permissions here:
+        https://docs-cortex.paloaltonetworks.com/r/Cortex-XSOAR/6.10/Cortex-XSOAR-Administrator-Guide/Automations
```
Comment (ShirleyDenkberg) [inline Packs/Core/Playbooks/playbook-Successful_guest_user_invitation.yml:993]:
```suggestion
        - For Cortex XSOAR 6, see https://docs-cortex.paloaltonetworks.com/r/Cortex-XSOAR/6.x/Cortex-XSOAR-Playbook-Design-Guide/Automations
        - For Cortex XSOAR 8 Cloud, see https://docs-cortex.paloaltonetworks.com/r/Cortex-XSOAR/8/Cortex-XSOAR-Cloud-Documentation/Create-a-script
       - For Cortex XSOAR 8 On-prem, see https://docs-cortex.paloaltonetworks.com/r/Cortex-XSOAR/8.7/Cortex-XSOAR-On-prem-Documentation/Create-a-script
```

File: Packs/Core/Playbooks/playbook-Successful_guest_user_invitation.yml [hunk 11]
```diff
@@ -0,0 +1,1604 @@
...
+    task:
+      id: d4e029ba-e082-48f7-8ea8-189f02abdbc9
+      version: -1
+      name: Get malicious IP value
+      description: |-
+        Set a value in context under the key you entered. If no value is entered, the script doesn't do anything.
+
+        This automation runs using the default Limited User role, unless you explicitly change the permissions.
+        For more information, see the section about permissions here:
+        https://docs-cortex.paloaltonetworks.com/r/Cortex-XSOAR/6.10/Cortex-XSOAR-Administrator-Guide/Automations
```
Comment (ShirleyDenkberg) [inline Packs/Core/Playbooks/playbook-Successful_guest_user_invitation.yml:1157]:
```suggestion
        - For Cortex XSOAR 6, see https://docs-cortex.paloaltonetworks.com/r/Cortex-XSOAR/6.x/Cortex-XSOAR-Playbook-Design-Guide/Automations
        - For Cortex XSOAR 8 Cloud, see https://docs-cortex.paloaltonetworks.com/r/Cortex-XSOAR/8/Cortex-XSOAR-Cloud-Documentation/Create-a-script
       - For Cortex XSOAR 8 On-prem, see https://docs-cortex.paloaltonetworks.com/r/Cortex-XSOAR/8.7/Cortex-XSOAR-On-prem-Documentation/Create-a-script
```

**Comment by ShirleyDenkberg** [comment]
@AdiPeret Doc review completed.

**Review by AdiPeret — APPROVED** [review]

**Pull request merged**